test/unit/testCheckAnswersSummaryJSONObjectBuilder.js

```diff
@@ -0,0 +1,148 @@
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Did the person who died leave a will?</dt>\n' +
+    '            <dd class="check-your-answers__answer">Yes</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Do you have the original will?</dt>\n' +
+    '            <dd class="check-your-answers__answer">Yes</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '    <h2 class="heading-medium">Inheritance tax</h2>\n' +
+    '    <dl class="check-your-answers check-your-answers--long">\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Has an Inheritance Tax (IHT) form been filled in?</dt>\n' +
+    '            <dd class="check-your-answers__answer">Yes</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">How was the Inheritance Tax (IHT) form submitted?</dt>\n' +
+    '            <dd class="check-your-answers__answer">By post</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '    <h2 class="heading-medium">The executors</h2>\n' +
+    '    <dl class="check-your-answers check-your-answers--long">\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">How many past and present executors are named on the will and any updates (&lsquo;codicils&rsquo;)?</dt>\n' +
+    '            <dd class="check-your-answers__answer">1</dd>\n' +
+    '        </div>\n' +
+    '<h3 class="heading-small">About you</h3>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">First name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Bobby</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Last name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Brown</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '    <h2 class="heading-medium">About the person who died</h2>\n' +
+    '    <dl class="check-your-answers check-your-answers--long">\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">First name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Graham</dd>\n' +
+    '        </div>\n' +
+    '        <div class="check-your-answers__row">\n' +
+    '            <dt class="check-your-answers__question">Last name(s)</dt>\n' +
+    '            <dd class="check-your-answers__answer">Greene</dd>\n' +
+    '        </div>\n' +
+    '    </dl>\n' +
+    '  </body>\n' +
+    '</html>\n';
```

Comment:
Instead of concatenating lines, why not use ES6 syntax with backticks?

Reply:
changed